src/components/deletePostModal/style.css

```diff
@@ -0,0 +1,30 @@
+.create-post-user-details {
+    display: grid;
+    grid-template-columns: 56px auto;
+    column-gap: 20px;
+}
+
+textarea {
```

Comment:
Be more specific with the selectors, this will target every `textarea` across the whole project and may cause errors